### PR TITLE
[TASK-12] [Test] As a user, I can logout the app and go back to the login screen

### PR DESCRIPTION
**Pull request by minhnimble**
https://github.com/minhnimble/android-surveys-list/projects/1#card-46433953

## What happened

Need to write unit tests for classes used in logout feature.
 
## Insight

**Acceptance Criteria**
- [x] Write unit tests
 
## Proof Of Work
No visible changes, just under the hood tests

### CODE REVIEW

**Comment by sleepylee** [comment]
I think I will review this again after you complete the rewrite on the other PR I had commented

**Review by manh-t — APPROVED** [review]

File: app/src/test/java/co/nimblehq/usecase/session/LogoutCompletableUseCaseTest.kt [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    fun setUp() {
+        mockRepository = mock()
+        useCase = LogoutCompletableUseCase(
+            TestRxSchedulerProviderImpl(),
+            mockRepository
+        )
+    }
+
+    @Test
+    fun `When logging out succeeds, it returns Complete`() {
```
Comment (sleepylee) [inline app/src/test/java/co/nimblehq/usecase/session/LogoutCompletableUseCaseTest.kt:29]: `successfully` - grammar 🙏

File: app/src/test/java/co/nimblehq/usecase/session/LogoutCompletableUseCaseTest.kt [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+            .test()
+
+        // Assert
+        positiveTestSubscriber
+            .assertNoErrors()
+            .assertComplete()
+    }
+
+    @Test
+    fun `When logging in fails, it returns a LogoutError`() {
```
Comment (sleepylee) [inline app/src/test/java/co/nimblehq/usecase/session/LogoutCompletableUseCaseTest.kt:47]: `logout failed`

**Review by sleepylee — APPROVED** [review]
some minor grammar

File: app/src/test/java/co/nimblehq/usecase/session/LogoutCompletableUseCaseTest.kt (hunk 1 above)
Comment (minhnimble) [reply app/src/test/java/co/nimblehq/usecase/session/LogoutCompletableUseCaseTest.kt:29]: will update 🙏

File: app/src/test/java/co/nimblehq/usecase/session/LogoutCompletableUseCaseTest.kt (hunk 2 above)
Comment (minhnimble) [reply app/src/test/java/co/nimblehq/usecase/session/LogoutCompletableUseCaseTest.kt:47]: will update 🙏

**Pull request merged**